dist/carousel/ds6/carousel.css

```diff
@@ -243,10 +245,15 @@ span.carousel__container {
 .carousel__control .icon--chevron-right,
 .carousel__control .icon--chevron-right-small {
   margin-left: 2px;
-  margin-top: -3px;
 }
 .carousel__control .icon--chevron-left,
 .carousel__control .icon--chevron-left-small {
   margin-left: -2px;
-  margin-top: -3px;
+}
+.carousel__control .icon--chevron-right,
```

Comment:
Can this group be reduced to just `.carousel__control .icon`?

Reply:
Removed in favor or Andrew's approach.